tests/Functional/ContentPublicationRenderTest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+use Drupal\Tests\oe_theme\PatternAssertions\FieldListAssert;
+use Drupal\user\Entity\Role;
+use Drupal\user\RoleInterface;
+
+/**
+ * Tests that our Publication content type renders correctly.
+ */
+class ContentPublicationRenderTest extends ContentRenderTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'system',
+    'oe_theme_helper',
+    'path',
+    'oe_theme_content_organisation',
+    'oe_theme_content_publication',
+    'oe_theme_content_entity_contact',
```

Comment:
We shouldn't enable dependencies on functional tests, the modules should do it. In this case we only need to list here `oe_theme_content_publication` and the rest should be enabled correctly by that module, via its dependencies. Same goes for the rest, `path`. Not sure why we need `block`?